example/models.py

```diff
@@ -86,3 +87,25 @@ class Comment(BaseModel):
 
     def __str__(self):
         return self.body
+
+
+class Project(PolymorphicModel):
```

Comment:
Nice use of the `Project` example. I see the same thing over in the `django-polymorphic` docs. That will be a nice help to those who are coming from viewing that documentation.